include/NovelRT/Transform.h

```diff
@@ -8,41 +8,74 @@
 #endif
 
 namespace NovelRT {
+  /**
+   * Represents a 2D position, scale and rotation.
+   */
   class Transform {
   protected:
     Maths::GeoVector2<float> _position;
     Maths::GeoVector2<float> _scale;
     float _rotation;
 
   public:
+    /**
+     * Creates a new Transform, with the specified position, rotation and scale.
+     * @param position The position, which represents the position where the transform will be rendered.
```

Comment:
Transforms can be used for other things besides rendering. Perhaps `which represents the position of the transform in a given space such as screen or world space.` or something similar would be more correct? Not sure I like the wording of my example, but hopefully my point makes sense.

Reply:
I saw that when looking at Unity docs too, just wasn't sure if there was still a world/screen space thing so i left that out. Just put that.